docs/experiments/shor.ipynb

```diff
@@ -547,6 +552,14 @@
     "        return ModularExp(target, exponent, base, modulus)\n",
     "\n",
     "    def apply(self, *register_values: int) -> int:\n",
+    "        \"\"\"Applies modular exponentiation to the registers.\n",
+    "\n",
+    "        Four registers should be passed in.  They are, in order:\n",
+    "          - the target\n",
+    "          - the exponent\n",
+    "          - the base\n",
+    "          - the modulus\n",
```

Comment:
It might be a little misleading to call base and modulus "registers" since they are constant parameters. These parameters control what quantum circuit is generated. The resulting circuit has two input registers: target and exponent. See also `make_order_finding_circuit`.